src/Common/test-runtime/project.json

```diff
@@ -16,7 +16,7 @@
   },
   "frameworks": {
     "dnxcore50": {
```

Comment:
Test infrastructure hasn't been updated yet.  @tarekgh and @karajas are working on that.